Modules/_io/iobase.c

```diff
@@ -83,25 +83,31 @@ iobase_unsupported(_PyIO_State *state, const char *message)
 _io._IOBase.seek
     cls: defining_class
     offset: int(unused=True)
-    whence: int(unused=True, c_default='0') = os.SEEK_SET
+      Offset as byte count.
+      Relative to the position given by 'whence'.
+    whence: int(unused=True, c_default='0') = io.SEEK_SET
+      Relative position, used by 'offset'. Valid values are:
+      * io.SEEK_SET -- Start of stream (the default)
+      * io.SEEK_CUR -- Current position
+      * io.SEEK_END -- End of stream
     /
 
-Change the stream position to the given byte offset.
+Change the stream position and return the new absolute position.
 
-The offset is interpreted relative to the position indicated by whence.
-Values for whence are:
+Offsets relative to the start of the stream are usually zero or
+positive, offsets relative to the current stream position may be
+zero, positive or negative, and offsets relative to the end of the
+stream are usually zero or negative.
 
-* os.SEEK_SET or 0 -- start of stream (the default); offset should be zero or positive
-* os.SEEK_CUR or 1 -- current stream position; offset may be negative
-* os.SEEK_END or 2 -- end of stream; offset is usually negative
+Some platforms allow seeking beyond the end of a file.
 
-Return the new absolute position.
+Note that not all file objects are seekable.
```

Comment:
We should include this admonition in the documentation for ``IOBase.seek`` (with reference to ``seekable()``?)

Reply:
Sounds good to me.